docs/source/_toctree.yml

```diff
@@ -53,6 +49,8 @@
     title: How to use 🤗 Accelerate with SageMaker
   - local: usage_guides/ipex
     title: How to use 🤗 Accelerate with Intel® Extension for PyTorch for cpu
+  - local: usage_guides/troubleshooting
+    title: Troubleshooting guide
```

Comment:
I'm not wondering if these may have more value in the Getting Started or Tutorials area. Or put this at the top of the table of contents for this section

Reply:
I'm ok with moving it into Tutorials, out of all the suggestions, it feels like the most fitting.